doc/index.rst

```diff
@@ -1,10 +1,44 @@
-The Plutus Platform
+Plutus Tools SDK User Guide
 ===============================
 
-The Plutus Platform is an application development platform for the Cardano blockchain.
-To read more about the Platform, see :ref:`what_is_the_plutus_platform`.
-To get started using the Platform see :ref:`plutus_getting_started`.
+Intended Audience
+-------------------------------
 
+External Developers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This user guide is intended for external developers who are authoring distributed 
+applications ("DApps") that implement smart contracts on the Cardano blockchain. 
+
+.. note::
+   If you are a developer who wants to contribute to the Plutus Tools SDK project, 
+   please refer to documentation residing in the `Plutus Tools SDK repository <https://github.com/
+   input-output-hk/plutus-apps>`_.
+
+Plutus Tools SDK
+===============================
+
+The Plutus Tools SDK is a collection of off-chain infrastructure resources built 
+for external developers. The `Plutus Tools SDK repository <https://github.com/
+input-output-hk/plutus-apps>`_ supports the underlying resources that developers 
+need who are writing full applications using Plutus in Haskell, including off-chain 
+code. The term “off-chain code” refers to the part of a contract application’s code 
+which runs off the chain. 
```

Comment:
```suggestion
code. The term “off-chain code” refers to the part of a contract application’s code 
which responds to events happening on the blockchain, usually by producing transaction. 
```
I am not sure about the correct, but the original text sounds a bit like a circular definition.

Reply:
This adds some useful perspective on what off-chain code is doing.

Reply:
@koslambrou @raduom In light of your suggestions, this is what I'm going with right now: 

> The term “off-chain code” refers to the part of a contract application’s code 
> which runs outside of the blockchain. Off-chain code responds to events happening 
> on the blockchain, usually by producing transactions.